app/models/source_file.rb

```diff
@@ -34,4 +34,8 @@ def claimed?
   def standards_import
     active_storage_attachment.record
   end
+
+  def pdf?
+    active_storage_attachment&.filename&.extension == "pdf"
```

Comment:
What about calling content type on the attachment? `active_storage_attachment.blob.content_type == "application/pdf"`